ggml-rpc.cpp

```diff
@@ -658,7 +643,7 @@ GGML_CALL ggml_backend_t ggml_backend_rpc_init(const char * endpoint) {
         }
     }
 #endif
-    GGML_PRINT_DEBUG("Connecting to %s\n", endpoint);
+    fprintf(stderr, "Connecting to %s\n", endpoint);
```

Comment:
nit: should it be better to use the `GGML_PRINT_DEBUG` instead of stdio directly?

Reply:
I think this message is helpful for troubleshooting connection problems, especially when using multiple `rpc-server`s, so I think it's fine to always print this